e2e/vendored_node/MODULE.bazel

```diff
@@ -0,0 +1,57 @@
+module(
+    name = "e2e_vendored_node",
+    version = "0.0.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "aspect_rules_js", version = "0.0.0")
+bazel_dep(name = "rules_nodejs", version = "5.5.3")
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "aspect_bazel_lib", version = "0.0.0")
+
+local_path_override(
+    module_name = "aspect_rules_js",
+    path = "../..",
+)
+
+# TODO: remove when http module extension released
+local_path_override(
+    module_name = "aspect_bazel_lib",
+    path = "../../../bazel-lib",
+)
+
+node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
+
+http = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "http")
+
+http.http_archive(
+    name = "vendored_node_linux_amd64",
```

Comment:
meh, I think leaving these in WORKSPACE.bzlmod is just as good for our current purposes and avoids the bazel-lib change being a blocker

Reply:
Good point. Updated.